Dockerfile

```diff
@@ -99,6 +102,17 @@ LABEL io.ceph.component="$NVMEOF_NAME" \
 
 ENV PYTHONPATH=$APPDIR/__pypackages__/$PYTHON_MAJOR.$PYTHON_MINOR/lib
 
+RUN \
+    --mount=type=cache,target=/var/cache/dnf \
+    --mount=type=cache,target=/var/lib/dnf \
+    dnf install -y python3-pip
+RUN \
+    --mount=type=cache,target=/root/.cache/pip \
+    TABULATE_INSTALL=lib-only pip install -U --target $APPDIR/__pypackages__/$PYTHON_MAJOR.$PYTHON_MINOR/lib tabulate
+RUN \
+    --mount=type=cache,target=/root/.cache/pip \
+    pip install -U --target $APPDIR/__pypackages__/$PYTHON_MAJOR.$PYTHON_MINOR/lib pyyaml
+
```

Comment:
not sure about new tabulate pyyaml pip dependencies.
in this project pdm is used to describe and install runtime dependencies.

Reply:
I followed the usage of other packages like setuptools which is installed using pip in Dockerfile. I'll try moving the packages to pyproject.toml.

Reply:
Moved.